salt/orch/kubernetes.sls

```diff
@@ -67,7 +67,7 @@ etcd_nodes_setup:
     - tgt: 'roles:etcd'
     - tgt_type: grain
     - highstate: True
-    - concurrent: True
+    - batch: 1
```

Comment:
Maybe we could make these parameters configurable, instead of hardcoding `1`.

We are currently not deploying _only-etcd_ nodes, but I think this could cause some problems if we create, for example, a 7-nodes etcd cluster. Provisioning would only happen in one machine at a time, and that would mean that masters would fail to find other master...